ONSdigital/eq-survey-runner
# Pip hashes

**_Changes**_
Pins all pypi dependencies to SHA256 hashes to stop downstream tampering. Also upgrade the version of pip on elastic beanstalk

**_How to test**_
Run a clean install using pip install -U -I -r requirements.txt
Edit one of the hashes in requirements.txt
Re-run the command pip install -U -I -r requirements.txt
It should fail, warning of a tampering with one of the packages.
Deploy to elastic beanstalk and check the application still works.

**_Who can test**_

Anyone but @dhilton and @warrenbailey
